scripts/inventory/infoblox.py

```diff
@@ -54,7 +55,7 @@ def main():
         if os.path.exists(config_file):
```

Comment:
```suggestion
        if os.path.exists(config_file):
```
I think `os.path.exists` crashes when being passed `None`.